scripts/validate-artifacts.ps1

```diff
@@ -1,121 +1,75 @@
-
-    if (-not $artifactsConfig.pricingCalculationsURL) { 
-        throw "Artifact config value 'pricingCalculationsURL' is empty! Please make sure that you executed the script 'scripts/generate-artifacts.ps1', and commited your changes"
-    } 
-    Invoke-WebRequest -Uri $artifactsConfig.pricingCalculationsURL -OutFile "$tempFolderPath/$spreadsheetName" -UseBasicParsing
-
+# initial validation
+Write-Output "Running initial validation"
+$context = Get-AzContext  
+if ($context)   
+{  
+    Write-Output "Azure Powershell module is installed, account is connected."  
+} else {  
+    throw "Please log in to Azure using Azure Powershell module (run Connect-AzAccount)"
+}  
+
+Write-Output "Checking if storage account exists"
+$storageAccount = (Get-AzStorageAccount -ErrorAction SilentlyContinue | Where-Object -Property 'StorageAccountName' -EQ -Value $ArtifactsStorageAccountName )
+if ($storageAccount) {
+    Write-Output "Storage account found"
+} else { 
+    throw "Unable to find storage account $ArtifactsStorageAccountName. Please make sure, that you specified the correct name of the storage account for the artifacts and that it is present in your Azure subscription"
 }
 
-Write-Output "Installing pre-requirements"
-Install-Module ImportExcel -Force 
-# Import-Module -Name ImportExcel
-
-
-Write-Output "Validating artifacts"
-
-$data = Import-Excel -StartRow 3 -Path "$tempFolderPath/$spreadsheetName" -WorksheetName $worksheetName
-
-$computeEstimate = $data | Where-Object {$_.'Service Category' -eq 'Compute'}
-if ($computeEstimate) { 
-    if ($computeEstimate.'Service Category'.Count -eq 1) { 
-        if ($computeEstimate.'Region' -ne 'UK West') { 
-            throw "Unable to verify the VM region - please make sure it set to UK West and try again."
-        } 
-
-        if (-not $computeEstimate.Description.Contains("B1s")) { 
-            throw "Unable to verify VM size in calulations - please make sure that VM size is set to B1s in the calculations and try again. "
-        } 
-        
-        if (-not $computeEstimate.Description.Contains("Linux")) { 
-            throw "Unable to verify VM OS type in calulations - please make sure that VM OS tyoe is set to Linux in the calculations and try again. "
-        } 
-
-        if (-not $computeEstimate.Description.Contains("1 managed disk – P4")) { 
-            throw "Unable to verify that you included OS disk to the calulations - please check and try again. "
-        } 
-        
-        Write-Output "`u{2705} Checked calculations for the Virtual Machine - OK."
-
-    } else { 
-        throw "Unable to verify estimates for the category 'Compute' - more than one VM found. Please make sure that you have only one VM in your calculations and try again."
-    }
-
+Write-Output "Checking if artifacts storage container exists" 
+$artifactContainer = Get-AzStorageContainer -Name $containerName -Context $storageAccount.Context -ErrorAction SilentlyContinue
```

Comment:
The `Get-AzStorageContainer` command also uses `-ErrorAction SilentlyContinue`. Similar to the previous comment, ensure that any errors are appropriately handled or logged.